recipes/cornac/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cornac" %}
+{% set version = "0.1.0.post4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4afaa2820aacfd174d207f386ef916fe156cbe89407a9e91b1d2f75684936755
+
+build:
+  skip: True  # [not linux or py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cython
```

Comment:
cython should also be a host dependency.

Reply:
updated!